cmd/cloud/server.go

```diff
@@ -357,6 +361,18 @@ var serverCmd = &cobra.Command{
 
 		eventsProducer := events.NewProducer(sqlStore, eventsDeliverer, awsClient.GetCloudEnvironmentName(), logger)
 
+		var cloudflareClient supervisor.Cloudflarer
+		if cloudflareToken := os.Getenv("CLOUDFLARE_API_TOKEN"); cloudflareToken != "" {
+			cfClient, err := cf.NewWithAPIToken(cloudflareToken)
+			if err != nil {
+				return errors.Wrap(err, "Failed to initialize cloudflare client using API token")
```

Comment:
nit:
```suggestion
				return errors.Wrap(err, "failed to initialize cloudflare client using API token")
```

There are a few more below as well.